### PR TITLE
Motion update with Exception

### PR DESCRIPTION
**Pull request by Johnobomb**
Closes: #9 
Corrected source code from previous pull request.

### CODE REVIEW

File: nav/stepper/stepper_constants.py [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
+BL_dir = 8
 
 # Fourth wheel GPIO Pin
-WHEEL4_step = 24
-WHEEL4_dir = 23
+FL_step = 24
+FL_dir = 23
+
+#Direction constants
+FWD = True
```
Comment (RishavRajendra) [inline nav/stepper/stepper_constants.py]: Talk to @BenjiLee0002 about this. All main constants should be in the same place for accessibility.

File: nav/stepper/stepper_constants.py [hunk 2]
```diff
@@ -1,26 +1,32 @@
...
+REV = False
 
 CW = 1 			                 # Clockwise rotation
 CCW = 0 		                 # Counter-clockwise rotation
 STEPS_PER_REVOLUTION = 800 		 # Steps per Revolution (360/1.8) * 4. Multiply by 4 because quarter step.
-
+DISTANCE = 60*math.pi/25.4
```
Comment (RishavRajendra) [inline nav/stepper/stepper_constants.py]: Same as #10, different wheels will have different formulas. Better structure needed.

File: nav/stepper/motion.py [hunk 3]
```diff
@@ -1,38 +1,76 @@
...
+                        GPIO.output(pin, CCW)
+                #else if REV    
+                elif dir == False:
+                    for pin in [BR_dir, FR_dir]:
+                        GPIO.output(pin, CCW)
+                    for pin in [BL_dir, FL_dir]:
+                        GPIO.output(pin, CW)
 
-step_count = STEPS_PER_REVOLUTION * 20
+                    for _ in range(distance):
```
Comment (RishavRajendra) [inline nav/stepper/motion.py]: Check out the main.py file of #10 for a better way to do this. We should not start and stop the motor controller from every function.

File: nav/stepper/motion.py [hunk 4]
```diff
@@ -1,38 +1,76 @@
...
+                        [GPIO.output(x, GPIO.HIGH) for x in self.wheels_step]
+                        sleep(delay)
+                        [GPIO.output(x, GPIO.LOW) for x in self.wheels_step]
+                        sleep(delay)
+                
+                    sleep(.5)    
+                    for pin in self.step_mode_select:
```
Comment (RishavRajendra) [inline nav/stepper/motion.py]: Unnecessary. Please remove.

File: nav/stepper/motion.py [hunk 5]
```diff
@@ -1,38 +1,76 @@
...
+            
+            try:
+                #if FWD
+                if dir == True:
+                    for pin in [BR_dir, FR_dir]:
+                        GPIO.output(pin, CW)
+                    for pin in [BL_dir, FL_dir]:
+                        GPIO.output(pin, CCW)
+                #else if REV    
+                elif dir == False:
```
Comment (BenjiLee0002) [inline nav/stepper/motion.py]: dir is a boolean value so you don't need to check if it is true or false, the if statement already checks for that.

if dir:
elif not dir:

**Pull request merged**